multi-agent/lib/mas/elements/sandboxes/common/card_builder.py

```
@@ -0,0 +1,29 @@
+
+A sandbox represents itself as a single skill.
+"""
+
+from mas.elements.common.card.models import ElementCard, Skill, CardBuildInput
+from mas.elements.common.card.interface import CardBuilder
+
+
+class SandboxCardBuilder(CardBuilder):
+    """Sandbox builds its card with itself as a skill."""
+
+    def build(self, input: CardBuildInput) -> ElementCard:
```

Ruff (0.15.20)

[error] Function argument `input` is shadowing a Python builtin

(A002)